tests/Nox.Types.Tests/CurrencyCode3Tests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Nox.Types.Tests.Types;
```

Comment:
Can we write the tests using FluentAssertions, as it was agreed before?

https://regusit.atlassian.net/wiki/spaces/NOX/pages/3153264693/Test+Projects